### PR TITLE
fixed review comments

### PR DESCRIPTION
**Pull request by nixxus1030**

### CODE REVIEW

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 1]
```diff
@@ -3152,40 +3321,38 @@ components:
     ReductionCardReferenceDef:
       type: object
       properties:
-        code:
+        cardValue:
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:3324]: Undos @CGantert345 patch :-(

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 2]
```diff
@@ -4008,20 +4152,20 @@ components:
...
-          description: >-
-            - MANDATORY: the booking can not be made without reservation
-            - INCLUDED: the reservation is included in the offer but is not mandatory
-            - OPTIONAL: the reservation is not included but can be integrated in the same ticket
-          type: string
-          x-extensible-enum:
-            - OPTIONAL
-            - MANDATORY
-            - INCLUDED
+                description: MANDATORY - the booking can not be made without reservation, INCLUDED - reservation is included in the offer but is not mandatory, OPTIONAL reservation is not included but can be integrated in the same ticket
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:4158]: Undos @schlpbch patch

**Review by schlpbch — CHANGES_REQUESTED** [review]

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ tags:
   - name: Offer Parts
     description: >-
       resources to manipulate parts of an offer consisting of, e.g., admissions, reservations,
-      or ancillaries 
+      integrated reservations or ancillaries
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:31]: My patch is still not reflected

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 4]
```diff
@@ -89,11 +83,10 @@ paths:
               - ANY_CODE
               - CODELIST
               - COORDS
-        - name: matchCodeList
+        - name: CodeListForMatch
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:86]: Lowercase please

**Review by schlpbch — APPROVED** [review]

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml (hunk 2 above)
Comment (CGantert345) [reply IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:4173]: What is cardValue?

**Review by schlpbch — APPROVED** [review]
Fine after having carefully merged with @nixxus1030

**Pull request merged**